livestyled/models/event_date.py

```diff
@@ -0,0 +1,21 @@
+class EventDate:
+    def __init__(
+            self,
+            id,
+            start_at,
+            end_at,
+            general_ticket_url
+    ):
+        self._id = id
+        self._start_at = start_at
+        self._end_at = end_at
+        self._general_ticket_url = general_ticket_url
```

Comment:
I know I made a bunch of member variables starting with underscores but in this scenario this wont work. I would suggest removing the leading underscore of everything apart from `self._id`